hack/slack/update_e2e_ondemand.py

```diff
@@ -0,0 +1,147 @@
+                "elements": [
+                    {"type": "text", "text": "result", "style": {"bold": True}}
+                ],
+            }
+        ],
+    },
+]
+
+
+rows = [table_header]
+
+with open("./hack/slack/slack-e2e-ondemand-payload.json") as f:
+    payload = json.load(f)
+
+
+with open("./hack/slack/slack-e2e-ondemand-payload.json", "w") as f1:
+    for k8s_env, ocp_env in zip(supported_k8s, supported_ocps):
+        k8s_env_clean = k8s_env.replace("-", "_").replace(".", "_").upper()
+        ocp_env_clean = ocp_env.replace("-", "_").replace(".", "_").upper()
+        row = [
+            {
+                "type": "rich_text",
+                "elements": [
+                    {
+                        "type": "rich_text_section",
+                        "elements": [{"type": "text", "text": k8s_env}],
+                    }
+                ],
+            },
+            {
+                "type": "rich_text",
+                "elements": [
+                    {
+                        "type": "rich_text_section",
+                        "elements": [
+                            {
+                                "type": "link",
+                                "url": "${{{{ env.{}_RUN_ID_URL }}}}".format(
+                                    k8s_env_clean
+                                ),
+                                "text": "tests ",
+                            },
+                            {
+                                "type": "emoji",
+                                "name": "${{{{ env.{}_EMOJI }}}}".format(k8s_env_clean),
+                            },
+                        ],
+                    }
+                ],
+            },
+            {
+                "type": "rich_text",
+                "elements": [
+                    {
+                        "type": "rich_text_section",
+                        "elements": [{"type": "text", "text": ocp_env}],
+                    }
+                ],
+            },
+            {
+                "type": "rich_text",
+                "elements": [
+                    {
+                        "type": "rich_text_section",
+                        "elements": [
+                            {
+                                "type": "link",
+                                "url": "${{{{ env.{}_RUN_ID_URL }}}}".format(
+                                    ocp_env_clean
+                                ),
+                                "text": "tests ",
+                            },
+                            {
+                                "type": "emoji",
+                                "name": "${{{{ env.{}_EMOJI }}}}".format(ocp_env_clean),
+                            },
+                        ],
+                    }
+                ],
+            },
+        ]
+
+        rows.append(row)
+
+    payload["blocks"][1] = {
+        "type": "table",
+        "rows": rows,
+    }
+    json.dump(payload, f1, indent=2)
```

Comment:
there is a `table_header` so maybe we could have file_header and file_trailer for mrkdwn/link elements instead of load/replace/dump approach? The file would be always created from scratch. It's contents would be more clear.